models/vtol_downwards_camera/vtol_downwards_camera.sdf

```diff
@@ -0,0 +1,24 @@
+<sdf version='1.5'>
+  <model name='vtol_downwards_camera'>
```

Comment:
The notion of camera is confusing as there are camera models in this repo that uses the `gst-camera-plugin`, which is a normal camera, not a depth camera. Could we possibly have a better name representing depth cameras?

Reply:
@Jaeyoung-Lim  do you have a naming suggestion?

Reply:
May be `vtol_depth_camera`? This is also consistent with the iris_depth_camera. I also realized this is also true for the `iris_downward_camera`